android-sdk/src/main/java/ai/evolv/android_sdk/helper/UtilityHelper.java

```diff
@@ -13,4 +21,26 @@
             return null;
         }
     }
+
+    public JsonElement setKeyToValue(String key, JsonElement value, JsonElement map){
+
+        JsonObject current = (JsonObject) map;
+        String[] keys = key.split(Pattern.quote("."));
+        for (int i = 0; i < keys.length; i++) {
+            String k = keys[i];
+            if (i == (keys.length - 1)) {
+                current.add(k,value);
```

Comment:
Does 'add' work is the key is already present?

Reply:
![photo_2021-06-03_11-57-28](https://user-images.githubusercontent.com/28364912/120617575-eb647300-c462-11eb-812b-a41c66a8041f.jpg)